src/var.cpp

```diff
@@ -1393,6 +1402,26 @@ VarState jitc_var_state(uint32_t index) {
         return VarState::Unevaluated;
 }
 
+VarInfo jitc_var_info(uint32_t index) {
+    VarInfo info;
```

Comment:
`VarInfo{}`? Otherwise this may return undefined output for some fields like `info.literal` or `info.data`